### PR TITLE
chore: add redirect for forgot password in spanish 

### PR DESCRIPTION
**Pull request by cbernat**
Add redirects for forgot password as follow:

- English: https://app.fromdoppler.com/#/login/reset-password
- Spanish: https://app.fromdoppler.com/#/ingresa/cambiar-clave

Reviewed this with @carodipietro 

Issue: DBR-266

https://cdn.fromdoppler.com/doppler-webapp/int-build622/#/login/reset-password
https://cdn.fromdoppler.com/doppler-webapp/int-build622/#/ingresa/cambiar-clave

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ class App extends Component {
             <PublicRouteWithLegacyFallback exact path="/forgot-password" />
             <RedirectWithQuery exact from="/ingresa" to="/login?lang=es" />
             <RedirectWithQuery exact from="/registrate" to="/signup?lang=es" />
+            <RedirectWithQuery exact from="/olvidaste-contrasenia" to="/forgot-password?lang=es" />
```
Comment (andresmoschini) [inline src/App.js]: Ugh!

Please, review it with @carodipietro, "contrasenia" is very ugly!

Comment (cbernat) [reply src/App.js]: I will do so :)

Comment (cbernat) [reply src/App.js]: Is not *that* ugly anyways.

Comment (cbernat) [reply src/App.js]: @carodipietro what do you think about this?

File: src/App.js [hunk 2]
```diff
@@ -94,6 +94,8 @@ class App extends Component {
             <PublicRouteWithLegacyFallback exact path="/forgot-password" />
             <RedirectWithQuery exact from="/ingresa" to="/login?lang=es" />
             <RedirectWithQuery exact from="/registrate" to="/signup?lang=es" />
+            <RedirectWithQuery exact from="/ingresa/cambiar-clave" to="/forgot-password?lang=es" />
+            <RedirectWithQuery exact from="/login/reset-password" to="/forgot-password?lang=en" />
```
Comment (andresmoschini) [inline src/App.js]: We have talked with @cbernat and decided using `/login/reset-password` as official URL and redirect `/forgot-password` to `/login/reset-password`.

**Comment by andresmoschini** [comment]
:shipit:

**Pull request merged**